src/redux/books/books.js

```diff
@@ -6,12 +6,33 @@ export const addedBook = (payload) => ({
   payload,
 });
 
-export const removedBook = (payload) => ({
+export const removedBook = (id) => ({
   type: REMOVE_BOOK,
-  payload,
+  id,
 });
 
-const books = [];
+const books = [
+  {
+    id: 1,
+    title: 'Think Big',
+    author: 'Ben Carson',
+  },
+  {
+    id: 2,
+    title: 'The Power of Significance',
+    author: 'John Maxwell',
+  },
+  {
+    id: 3,
+    title: 'Change your thinking cahnge your life',
+    author: 'Brian Tracy',
+  },
+  {
+    id: 4,
+    title: 'The Leader who had no title',
+    author: 'Robin Sharma',
+  },
+];
```

Comment:
- [ ] Please fix the remove book functionality because it's not working properly. When I try to remove any initially added books nothing happened but when I remove any newly added book it's working just fine.

![chrome-capture-2022-6-21](https://user-images.githubusercontent.com/68796232/180106744-acbcde78-2979-4354-b2d0-82b29c0286e5.gif)

Reply:
done